trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/generate_propeller_wake_distribution.py

```diff
@@ -68,13 +66,13 @@ def generate_propeller_wake_distribution(prop,thrust_angle,m,VD,init_timestep_of
     wake_skew_angle = np.arctan(mu_prop/lambda_tot)
 
     # reshape gamma to find the average between stations 
-    gamma_new = np.zeros((m,Na,(Nr-1)))
-    gamma_new = (gamma[:,:,:-1] + gamma[:,:,1:])*0.5
+    gamma_new = np.zeros((m,Na,(Nr-1))) #[control points, azimuth  , radial stations -1 ] one less because ring
+    gamma_new = (gamma[:,:,:-1] + gamma[:,:,1:])*0.5 #  not sure if correct 
```

Comment:
@mclarke2 can you be sure if this is correct?

Reply:
I averaged the circulations along the propeller blade to reduce the dimension by 1 so its an approximation. The reason for this is that I use the radial stations to define the distributed vortex elements, i,e, if they are 20 stations, they are 19 vortex rings spanwise. The other alternative that I looked into was increasing the discretization of the propeller to 21 allowing me to retain the correct dimension of gamma - this lead to an issue in the slipstream contraction function which also reduced the dimension of the wake by 1 i.e. 19 - as a result, I opted to do an average between stations.